core/src/mindustry/mod/ContentParser.java

```diff
@@ -588,6 +601,12 @@ ContentType.status, parser(ContentType.status, StatusEffect::new),
                 value.remove("sector");
                 value.remove("planet");
 
+                if(value.has("rules")){
+                    JsonValue r = value.remove("rules");
+                    if(!r.isObject()) throw new RuntimeException("Rules must be an object!");
+                    out.rules = rules -> readFields(rules, r); //TODO is there a way to make it only need to read fields once?
+                }
+
```

Comment:
Should the same be done with rules?

Reply:
No, since rules are shallow copy, and performance is less important.

Reply:
Wait... this means that clones of that one controller will all share the same timer instance, leading to bizarre bugs.

Reply:
aaaaaAAAAAAAA